app/src/main/java/org/wikipedia/talk/TalkTopicsActivity.kt

```diff
@@ -128,7 +131,12 @@ class TalkTopicsActivity : BaseActivity() {
                 }
                 .subscribe({ response ->
                     topics.clear()
-                    topics.addAll(response.topics!!)
+                    for (topic in response.topics!!) {
+                        if (topic.id == 0 && topic.html!!.trim().isEmpty()) {
```

Comment:
Since we are going to hide the first section of the talk page, should we show the `talkEmptyContainer` view if there are no other topics?

Reply:
Good call. Done